### ISSUE TITLE
Allow to customize Json serialization configuration

### ISSUE THREAD

**Issue by kormik** [rjaros/kvision#324]
Serialization is used in couple of places in KVision and I would like to have an option to configure serializer for the whole project.
For example Rest client and Forms usually works with same data classes, but each create own instance of Json and does not allow to customize it.

I can imagine an object with default Json config which could be overriden by user and all KVision components use it:

```
object Mapper {
    val instance = Json {
        ignoreUnknownKeys = true
        isLenient = true
        encodeDefaults = true
    }
    
    fun init(customJson:Json) {
        instance = customJson
    }
```

**Comment by rjaros** [comment]
This will be possible in the next version. The custom configuration will be used by forms, both rest clients and tabulator. Standard `DateSerializer` will be included automatically.
```
        import io.kvision.utils.Serialization

        Serialization.customConfiguration = Json {
            ignoreUnknownKeys = true
            isLenient = true
            encodeDefaults = true
            serializersModule = SerializersModule {
                polymorphic(Entity::class) {
                    subclass(Application::class)
                    subclass(Asset::class)
                    subclass(DialogueMixin::class)
                    subclass(File::class)
                    subclass(Voice::class)
                }
            }
            classDiscriminator = "__class"
        }
```

**Comment by rjaros** [comment]
Released with 5.7.1
